### PR TITLE
Re-write Dataflow > Enrich in an attempt to improve clarity

### PR DESCRIPTION
**Pull request by ga-hegsvold**

### CODE REVIEW

**Review by BaardBouvet — APPROVED** [review]

File: sesam-dataflow.rst [hunk 1]
```diff
@@ -66,26 +66,73 @@ Sesam has a built-in monitoring function to help to ensure data flows as expecte
 Enrich
 ------
 
-The enrichment step is concerned with adding :ref:`namespaces <concepts-namespaces>` and :ref:`namespaced identifiers <best-practice-namespace>`. Sesam makes use of RDF (https://www.w3.org/RDF/) to apply this enrichment and as such describe what a certain field means. Other types of semantics can be utilized if need be.
+The enrichment step is concerned with providing sematics in three main areas:
```
Comment (grove) [inline sesam-dataflow.rst:69]: typo: sematics

File: sesam-dataflow.rst [hunk 2]
```diff
@@ -66,26 +66,73 @@ Sesam has a built-in monitoring function to help to ensure data flows as expecte
...
 
-The following are benefits of semantic enrichment:
+Regardless, make sure the NIs reference actual entity identifiers (primary key equivalents) in the related datasets.
 
-- **References to other datasets**: if a property is a reference or relation to another dataset, such as a foreign key field in a relational database, you should add an additional property that contains a reference to that dataset. This should be in the form of a :ref:`namespaced identifier <best-practice-namespace>`. These references are usually key properties when semantically linking data together in a global dataset, :ref:`connect`, but are also useful when connecting data in preparation pipes, :ref:`transform`.
+.. important::
 
-    .. hint::
+  NIs should only reference business entities from the *same* source,
+  the same way FKs in relational databases references primary keys in tables within the *same* database.
```
Comment (grove) [inline sesam-dataflow.rst:129]: I think it is better to spell FKs out as "foreign keys".

File: sesam-dataflow.rst (hunk 1 above)
Comment (ga-hegsvold) [reply sesam-dataflow.rst]: fixed

File: sesam-dataflow.rst (hunk 2 above)
Comment (ga-hegsvold) [reply sesam-dataflow.rst]: fixed

**Review by grove — APPROVED** [review]

**Pull request merged**